guidelines/content-guidelines/screenshots.md

```diff
@@ -0,0 +1,46 @@
+# Screenshots
+
+As someone once said, a picture is worth a thousand words. Therefore, whenever you need to illustrate operations performed on the UI, feel free to use a screenshot to convey a lot of information visually.
+
+Follow these basic principles when placing a screenshot in your content:
+- Do not overuse screenshots and limit  visual noise.
+- Do not use directional indicators such as "above" and "below" to refer to screenshots. Instead, include a brief introduction before each screenshot that describes its purpose and any necessary details.
+- Do not include the mouse pointer in your screenshots, unless it shows a function related to the content.
+- Do not include unnecessary elements, such as the browser toolbar. Include only elements that are relevant in a matter of a given content.
+
+For details on how to format screenshots and their elements in the Kyma documents, see the particular document sections.
+
+## Tool
+
+Adjust or capture your screenshots using any tool that outputs high quality images, such as [Snagit](https://www.techsmith.com/screen-capture.html). The desired image format is SVG, but PNG and JPG formats are also acceptable.
+Use an online tool such as [TinyPNG](https://tinypng.com/) to compress images and limit the size of each image to 1MB, or smaller.
+If you want to control the size of the image relative to the screen size, use one of these standard percentages: 100%, 75%, 50%, or 25%.
+
+> **NOTE:** Name the file as `{screenshot-name}` and save it under the corresponding `assets` directory.
+
+## Borders
+
+Use **grey** (HEX: #D2D5D9) 1pt border for the screenshot.
+
+## Steps
+
+If necessary, mark multiple areas or steps on the screenshot using **blue** (HEX: #0A6ED1) round stamps with white numbers.
+Explain the steps under the screenshot with the ordered list.
+
+## Indicators
+
+To highlight a certain area of your screenshot, use **red** (HEX: #EF27270) 10pt arrow or box.
```

Comment:
arrow or box -> `for arrows or boxes`or `use a **red** (HEX: #EF27270) 10pt arrow or a box.`